zerver/openapi/zulip.yaml

```diff
@@ -5019,6 +5019,14 @@ paths:
 
         `{msg_id}` in the above URL should be replaced with the ID of the
         message you wish you update.
+        ## Permissions
```

Comment:
I think we can skip this heading, unless there's a link to it.  Also, this information is probably inaccurate; we should instead just link to the /help/ article on message editing permissions: https://zulip.com/help/configure-message-editing-and-deletion.  So maybe we just have a sentence:

`See [configuring message editing](/help/configure-message-editing-and-deletion) for detailed documentation on under what circumstances users are allowed to edit topics."

Reply:
Made the appropriate modifications.